developer_manual/design/index.rst

```diff
@@ -5,6 +5,7 @@ Design guidelines
 .. toctree::
    :maxdepth: 2
 
+   newbutton
```

Comment:
I'd say this should be part of the navigation section. :wink:

Reply:
I asked myself the same question! I think you're right :)

Reply:
Why is it placed outside? Then it wouldn't be in the sidebar anymore.

Reply:
Hu! I'm an idiot, I wanted to say outside the ul navigation, but yes, this is inside the app-navigation. ^^'

Reply:
Nevertheless, we also have a specific settings page which is also in the app-navigation div, so which do you prefer? Inside the navigation page or like this ?

Reply:
Ah ok :wink: Makes sense to have them grouped by the "app-navigation". Maybe we should also move the settings stuff to there, since that is in the app-navigation as well.

Reply:
I agree! :)